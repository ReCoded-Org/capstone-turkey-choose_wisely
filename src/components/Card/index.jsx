import React from "react";
import { Row, Col, Container } from "react-bootstrap";
import "./style.scss";
import Avatar from "@material-ui/core/Avatar";

export default function SliderSecion(speech) {
  return (
    <Container>
      <Row className="align-items-center justify-content-center Card">
        <Col className="ps-5 avatarComponent" md={2}>
          <Avatar className="Avatar" alt="Remy Sharp" />
        </Col>

        <Col className="col-md-offset-2 SpeechComponent" md={10}>
          <p className="paragraph">{speech.speechText}</p>
          <h5 className="auther">{speech.auther}</h5>
          <h6 className="description">{speech.AutherSection}</h6>
        </Col>
      </Row>
    </Container>
  );
}
