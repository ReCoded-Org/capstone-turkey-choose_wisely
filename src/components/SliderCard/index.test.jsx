import React from "react";
import Card from "./";
import renderer from "react-test-renderer";

jest.mock("react-i18next", () => ({
  useTranslation: () => ({ t: (key) => key }),
}));

const speech = {
  speechText1:
    "Currently, Ford sells six sedans and coupes in North America with the Fiesta, Focus, Fusion, C-Max, Mustang and Taurus. This lineup hits multiple segments from the compact Fiest to the mid-size Focus, C-Max and Fusion to the full-size Taurus",
  auther1: " Emily JÃ¸rgense",
  AutherSection1: " Author at Panoply Store",
};
test("Snapshop test of Card component", () => {
  const map = renderer.create(<Card {...speech} />);
  let tree = map.toJSON();

  expect(tree).toMatchSnapshot();
});
