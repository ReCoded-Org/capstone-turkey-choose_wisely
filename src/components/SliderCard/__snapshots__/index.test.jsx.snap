// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshop test of Card component 1`] = `
<div
  className="container"
>
  <div
    className="align-items-center justify-content-center Card row"
  >
    <div
      className="ps-5 avatarComponent col-md-2 col-sm-12"
    >
      <div
        className="MuiAvatar-root MuiAvatar-circular Avatar MuiAvatar-colorDefault"
      >
        <svg
          aria-hidden={true}
          className="MuiSvgIcon-root MuiAvatar-fallback"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 12c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0 2c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4z"
          />
        </svg>
      </div>
    </div>
    <div
      className="col-md-offset-2 SpeechComponent col-md-10 col-sm-12"
    >
      <p
        className="paragraph"
      />
      <h5
        className="auther"
      />
      <h6
        className="description"
      />
    </div>
  </div>
</div>
`;
